prime-angular-frontend/src/app/shared/components/pages/page-footer/page-footer.component.html

```diff
@@ -13,7 +13,7 @@
   <div class="col pt-2 text-right">
 
     <button mat-flat-button
-            type="button"
+            type="submit"
```

Comment:
You'll want to make sure this doesn't affect other workflows since this is a shared component.

Reply:
tested successfully enrolment, site and admin HA site and paper enrolment workflows